### ISSUE TITLE
Minimal config doesn't build

### ISSUE THREAD

**Issue by kai687** [kai687/sphinxawesome-theme#947]
**This config**

```
project = "Test"
exclude_patterns = ["_env"]
extensions = ["sphinxawesome_theme"]
html_theme = "sphinxawesome_theme"
```

leads to **this error**

```
Failed to copy a file in html_static_file: /Users/welke.kai/Desktop/test/_env/lib/python3.10/site-packages/sphinxawesome_theme/static/docsearch_config.js_t: UndefinedError("'docsearch_config' is undefined")
```

**Comment by kai687** [comment]
Is already fixed in the master branch. Need to publish a new version.

**Comment by kai687** [comment]
Published 3.3.1
